contributors/design-proposals/customresources-subresources.md

```diff
@@ -131,17 +90,32 @@ type ScaleStatus struct {
 The `SubResources` field in `CustomResourceDefinitionSpec` will be gated under the `CustomResourceSubResources` alpha feature gate.
 If the gate is not open, the value of the new field within `CustomResourceDefinitionSpec` is dropped on creation and updates of CRDs.
 
+### Scale type
+
+The `Scale` object is the payload sent over the wire for `/scale`. The [polymorphic `Scale` type](https://github.com/kubernetes/kubernetes/pull/53743) is used for the `Scale` object.
+
 ## Semantics
 
 ### Validation Behavior
 
 #### Status
 
 The status endpoint of a CustomResource receives a full CR object. Changes outside of the `.status` subpath are ignored.
-For validation everything outside of the `.status` subpath can be reset to the existing values of the CustomResource. Then the JSON Schema presented in the CRD is validated against the whole object.
+For validation, the JSON Schema present in the CRD is validated only against the `.status` subpath.
+
+To validate only against the schema for the `.status` subpath, `oneOf` and `anyOf` constructs are not allowed within the root of the schema
+i.e. the following is forbidden in the CRD spec:
+
+```yaml
+validation:
+    openAPIV3Schema:
+        oneOf:
+            ...
+```
 
-Note: one could extract the status subobject, filter the JSON schema by rules applied to the status and then validate the status subobject only.
-The filter step of the JSON schema though is not obvious in the case that there are status validations not below a properties JSON Schema construct. For that reason, the alternative implementation is preferred as its semantics are much simpler.
+A function `projectJSONSchema(Schema, path []string) (Schema, error)` can be used to project the schema in the CRD
```

Comment:
wondering if this comes off as an implementation detail...

Reply:
Yes, maybe. But as a note for better understanding it does not harm.